### PR TITLE
Add protocol version for 1.20.1

### PR DESCRIPTION
**Pull request by NoltoxGit**

### CODE REVIEW

**Comment by NoltoxGit** [comment]
Fixe #2457

File: src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public final class MinecraftVersion implements Comparable<MinecraftVersion>, Ser
     /**
      * Version 1.20 - the trails and tails update
      */
-    public static final MinecraftVersion TRAILS_AND_TAILS = new MinecraftVersion("1.20");
+    public static final MinecraftVersion TRAILS_AND_TAILS = new MinecraftVersion("1.20.1");
```
Comment (dmulloy2) [inline src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java:43]: if 1.20.1 requires special handling, it should be a new field. in any case, this should stay 1.20

**Comment by NoltoxGit** [comment]
When I fix it, I get the alert back in my console :/

File: src/main/java/com/comphenix/protocol/utility/MinecraftProtocolVersion.java [hunk 2]
```diff
@@ -88,6 +88,7 @@ private static NavigableMap<MinecraftVersion, Integer> createLookup() {
         map.put(new MinecraftVersion(1, 19, 4), 762);
 
         map.put(new MinecraftVersion(1, 20, 0), 763);
+        map.put(new MinecraftVersion(1, 20, 1), 764);
```
[the remark this answers is not in the document]
Comment (NoltoxGit) [reply src/main/java/com/comphenix/protocol/utility/MinecraftProtocolVersion.java:91]: > Protocol must be "763".

I will try this

**Comment by NoltoxGit** [comment]
Okay i see, il will fix all

**Review by Kichura — APPROVED** [review]
This part looks good.

**Pull request closed without merging**